pkgs/development/python-modules/playwright/driver.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchzip
+, playwrightVersion
+}:
+
+stdenv.mkDerivation rec {
+  name = "playwright-driver";
+  src =
+    let
+      inherit (stdenv.hostPlatform) system;
+      selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
+      suffix = selectSystem {
+        x86_64-linux = "linux";
+        aarch64-linux = "linux-arm64";
+        x86_64-darwin = "mac";
+        aarch64-darwin = "mac-arm64";
+      };
+      sha256 = selectSystem {
+        x86_64-linux = "sha256-RyTYrIBMIFuIfr/WRH+qXf3DcQvEDLBePD1PeKzY93c=";
+        aarch64-linux = "sha256-u5OtIVygcVQALqxWO/bEhf7jaGC6PtY/1sqrb5GH/kM=";
+        x86_64-darwin = "sha256-KTg4nt/MZL7PzBCsU0Y9Pq9j2QEbG3iOdhWdC66romQ=";
+        aarch64-darwin = "sha256-xdrWn+fZC0QbUe3uCqWErtTaSKWOIwp+0zLLSFXnA/A=";
+      };
+    in
+    fetchzip {
+      url = "https://playwright.azureedge.net/builds/driver/playwright-${playwrightVersion}-${suffix}.zip";
+      inherit sha256;
+      postFetch = ''
+        mkdir -p $out/driver
+        install $downloadedFile $out/driver/playwright-${playwrightVersion}-${suffix}.zip
+      '';
+    };
+  installPhase = ''
+    mkdir -p $out/driver
+    cp -r driver/* $out/driver
+  '';
+}
```

Comment:
Since you're not actually unarchiving, this can be simplified a bit. You don't even need to wrap it in another `mkDerivation`:

```suggestion
let
  inherit (stdenv.hostPlatform) system;
  selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
  suffix = selectSystem {
    x86_64-linux = "linux";
    aarch64-linux = "linux-arm64";
    x86_64-darwin = "mac";
    aarch64-darwin = "mac-arm64";
  };
  sha256 = selectSystem {
    x86_64-linux = "sha256-RyTYrIBMIFuIfr/WRH+qXf3DcQvEDLBePD1PeKzY93c=";
    aarch64-linux = "sha256-u5OtIVygcVQALqxWO/bEhf7jaGC6PtY/1sqrb5GH/kM=";
    x86_64-darwin = "sha256-KTg4nt/MZL7PzBCsU0Y9Pq9j2QEbG3iOdhWdC66romQ=";
    aarch64-darwin = "sha256-xdrWn+fZC0QbUe3uCqWErtTaSKWOIwp+0zLLSFXnA/A=";
  };
in
fetchurl {
  name = "playwright-driver";

  url = "https://playwright.azureedge.net/builds/driver/playwright-${playwrightVersion}-${suffix}.zip";
  inherit sha256;
  recursiveHash = true;

  downloadToTemp = true;

  postFetch = ''
    mkdir -p $out/driver/
    install "$downloadedFile" $out/driver/playwright-${playwrightVersion}-${suffix}.zip
  '';
}
```

Note that the hashes should *probably* not change because the recursive calculation mode should yield the same results, unless I'm missing something here.